christfellowshipchurch/web-app-v2
# Fixed Wistia in Events

### About
This PR allows wistia videos to work on Event Pages as well.

### Test Instructions
1. Run the Web App and go to [events/promotional-video](http://localhost:3000/events/promotional-video)
2. You should see the Wistia Video Player even if there is a video link in the promotional video section in Rock, [here](https://rock.christfellowship.church/page/2798?ContentItemId=14811).
3. Test to make sure a video is still displayed when no Wistia Media is attached and just a video link is provided in Rock.

### Screenshots
<img width="1440" alt="Screenshot 2023-09-06 at 9 05 16 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/a6721b11-89fb-4b79-8156-2ede56444bd2">


### Closes Tickets
[CFDP-2714](https://christfellowshipchurch.atlassian.net/browse/CFDP-2714)


[CFDP-2714]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2714?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
